app/dao/fact_notification_status_dao.py

```diff
@@ -258,7 +269,18 @@ def fetch_notification_status_for_service_for_today_and_7_previous_days(service_
             Notification.notification_type.cast(db.Text),
             Notification.status,
             *([Notification.template_id] if by_template else []),
-            func.count().label("count"),
+            *(
+                [
+                    case(
+                        [
+                            (Notification.notification_type == "email", func.count()),
+                        ],
+                        else_=func.sum(Notification.billable_units),
+                    ).label("count")
+                ]
+                if current_app.config["FF_SMS_PARTS_UI"]
+                else [func.count(Notification.id).label("count")]
```

Comment:
I think `func.count(Notification.id).label("count")` is the same as `func.count().label("count")` - any particular reason for the change in the FF off case?

Reply:
Hmm I think I just explicitly typed this out, probably confused myself with all of this horrid syntax :)

It SHOULD be the same, but I'll set it back to how it was exactly in the else case.  Good eye!
```suggestion
                else [func.count().label("count")]
```